### PR TITLE
Add notes on BAAPAM

### PR DESCRIPTION
**Pull request by frcroth**

### CODE REVIEW

**Comment by frcroth** [comment]
Maybe we can also take a new picture that shows using an authenticator on a lock

File: profile/README.md [hunk 1]
```diff
@@ -40,3 +42,22 @@ The dependencies we used did not support the newest FIDO2 features, so we added
 - [python-keycloak](https://github.com/All-Your-Locks-Are-Belong-To-Us/python-keycloak/tree/fix/fetch-token-for-client)
 - [webauthn-json](https://github.com/All-Your-Locks-Are-Belong-To-Us/webauthn-json/tree/feature/credential-protection-policy)
 - [py_webauthn](https://github.com/All-Your-Locks-Are-Belong-To-Us/py_webauthn/tree/feature/credential-protection-policy)
+
+## Implementing a FIDO2 Relying Party on microcontrollers
+
+In the 2022 summer semester we built upon the previous project and **implemented a FIDO2 library for microcontrollers called [libmicrofido2](https://github.com/All-Your-Locks-Are-Belong-To-Us/libmicrofido2)**.
+
+The library was inspired by [libfido2](https://github.com/Yubico/libfido2/) and we ran it on the AVR ATmega1284P, the nRF52480 and the ESP32-C3FN4 microcontrollers.
```
Comment (sirkrypt0) [inline profile/README.md:50]: I think [AVR is only the family](https://en.wikipedia.org/wiki/AVR_microcontrollers) and we therefore don't need to name it explicitly.

```suggestion
The library was inspired by [libfido2](https://github.com/Yubico/libfido2/) and we ran it on the ATmega1284P, the nRF52480 and the ESP32-C3FN4 microcontrollers.
```

**Review by sirkrypt0 — CHANGES_REQUESTED** [review]

**Comment by sirkrypt0** [comment]
> Maybe we can also take a new picture that shows using an authenticator on a lock

Good idea!

File: profile/README.md (hunk 1 above)
Comment (felix-gohla) [reply profile/README.md:50]: Also, we could move this before the other part about HotSIR.

**Review by felix-gohla — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (frcroth) [reply profile/README.md:50]: Does it still make sense that way? Since this term is very much building on the previous stuff.

Comment (felix-gohla) [reply profile/README.md:50]: Actually, I don't care. If you mean it wouldn't make sense for the latest stuff to be at the top, then you can leave it as it is.

**Review by felix-gohla — APPROVED** [review]

**Review by sirkrypt0 — APPROVED** [review]

**Pull request merged**